Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityProfileCloneWindow.cs

```diff
@@ -0,0 +1,307 @@
+﻿using Microsoft.MixedReality.Toolkit.Core.Definitions;
+using Microsoft.MixedReality.Toolkit.Core.Extensions.EditorClassExtensions;
+using System.Collections.Generic;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Inspectors.Profiles
+{
+    public class MixedRealityProfileCloneWindow : EditorWindow
+    {
+        private struct SubProfileAction
+        {
+            public enum TypeEnum
```

Comment:
Not overly fond of the enum name. TypeEnum doesn't describe the usage. Perhaps "ProfileCloneBehavior"?

Reply:
Also, does it make sense to have a leave empty option for sub-profiles that the user does _not_ want to use?

Reply:
Hadn't thought of that. I'll add it.